package.json

```diff
@@ -75,15 +49,46 @@
   "license": "MIT",
   "lint-staged": {
     "*.{js,jsx,json,md,ts,tsx}": [
-      "prettier --write",
-      "git add"
+      "prettier --write"
     ],
     "*.{js,jsx}": "npm run test:lint:js",
     "*.{ts,tsx}": "npm run test:lint:ts",
     "*.{md}": "npm run test:lint:md"
   },
   "main": "index.js",
   "name": "@encoura/eslint-config",
+  "overrides": {
+    "rimraf": "^6.0.1",
+    "test-exclude": "^7.0.0"
+  },
+  "peerDependencies": {
```

Comment:
I don’t recommend moving a large number of dependencies into peerDependencies because it shifts the burden of version management to consumers of the library, making integration more difficult. It also increases the risk of version mismatches and runtime issues, especially if consumers use versions that differ from what the library expects. This will create a lot of new noise with dependabot across the GitHub org. Thoughts?